source/includes/_payment-transaction.md

```diff
@@ -126,10 +154,13 @@ TODO
 
 **Query Parameters**
 
-| Name | Type | Required | Default | Description |
-| ---- | -----| -------- | ------- | ----------- |
-| **withPluginInfo** | boolean | no | false | If true, include plugin info |
-| **withAttempts** | boolean | no | false | if true, include payment attempts |
+| Name | Type | Required | Default | Description                      |
+| ---- | -----| -------- | ------- |----------------------------------|
+| **withPluginInfo** | boolean | no | false | If true, include plugin info. This results in the [PaymentPluginApi.getPaymentInfo](https://github.com/killbill/killbill-plugin-api/blob/cd3fb251c4b61931eb0f7954037145148d62f983/payment/src/main/java/org/killbill/billing/payment/plugin/api/PaymentPluginApi.java#L144) method being invoked for the underlying payment plugin |
```

Comment:
Would it make sense instead to link to our payment plugin doc, and specifically to [this section](https://docs.killbill.io/latest/payment_plugin#paymentpluginapi_methods) where we have a small description of `getPaymentInfo` ?